### PR TITLE
Added geometry<->WKT coder/decoder

### PR DESCRIPTION
**Pull request by Abyss777**
There are coder and decoder.

They assume common `lat` `lon` order in `area`

`wktToGeometry` can be used further to draw geofences on init and to draw on edit geofence in `GeofenceMap`

### CODE REVIEW

File: web/app/GeofenceConverter.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.GeofenceConverter', {
+    singleton: true,
+
+    wktToGeometry: function (mapView, wkt) {
+        var geometry, points = [];
```
Comment (tananaev) [inline web/app/GeofenceConverter.js]: Please combine all variable into one definition.

File: web/app/GeofenceConverter.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    radius = (Number(tmp[2]) / ol.proj.METERS_PER_UNIT.m) * resolutionFactor;
+                    geometry = new ol.geom.Circle(center, radius);
+                }
+            }
+        }
+        return geometry;
+    },
+
+    geometryToWkt: function (projection, geometry) {
+        var result;
```
Comment (tananaev) [inline web/app/GeofenceConverter.js]: Same, combine all variable into one definition.

File: web/app/GeofenceConverter.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ */
+
+Ext.define('Traccar.GeofenceConverter', {
+    singleton: true,
+
+    wktToGeometry: function (mapView, wkt) {
+        var geometry, points = [];
+        var projection, resolutionAtEquator, pointResolution, resolutionFactor;
+        var center, radius;
+        var m, i, lat, lon, tmp;
```
Comment (tananaev) [inline web/app/GeofenceConverter.js]: It would be nice to have more descriptive names for `m` and `tmp`.

**Comment by Abyss777** [comment]
Done

**Comment by tananaev** [comment]
Merged, thanks. Does geofence saving work?

**Comment by Abyss777** [comment]
Yes, sure.

**Pull request merged**